rest/src/test/java/com/redhat/runtimes/inventory/web/TestLifecycleManager.java

```diff
@@ -0,0 +1,53 @@
+/* Copyright (C) Red Hat 2023 */
+package com.redhat.runtimes.inventory.web;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import io.smallrye.reactive.messaging.memory.InMemoryConnector;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+/**
+ * This class is based on TestLifecycleManager from RedHatInsights/notifications-backend:
+ * https://github.com/RedHatInsights/notifications-backend/blob/master/engine/src/test/java/com/redhat/cloud/notifications/TestLifecycleManager.java
+ */
+public class TestLifecycleManager implements QuarkusTestResourceLifecycleManager {
+
```

Comment:
Is this class actually referenced from anywhere?

Reply:
It's used as a QuarkusTestResource for the test class at: https://github.com/RedHatInsights/insights-runtimes-inventory/blob/55c6847fa974b797a16027d32e55debae865af93/rest/src/test/java/com/redhat/runtimes/inventory/web/DisplayInventoryTest.java#L19